lib/helper/Playwright.js

```diff
@@ -520,7 +520,15 @@ class Playwright extends Helper {
 
     // close other sessions
     try {
-      const contexts = await this.browser.contexts();
+      let contexts = [];
+
+      // when using userDataDir, this.browser is now BrowserContext.
```

Comment:
Sorry for adding a comment suddenly but I have a question.
At line 543, this method return `this.browser` object, does this also work for BrowserContext for the further test scenarios?

Reply:
Thanks for checking the comment @kobenguyent  I faced this error at the beginning of the 2nd scenario execution if there are multiple scenarios. I tried to execute it sequentially.

`    [1] Error | Error: browserContext.newPage: Target page, context or browser has been closed
Error: browserContext.newPage: Target page, context or browser has been closed`

Reply:
Sorry I am wrong but yes this is because,
before the patch, I got this as same as original issue.
`TypeError: this.browser.contexts is not a function
    at Playwright._after (/node_modules/codeceptjs/lib/helper/Playwright.js:522:43)
    at /node_modules/codeceptjs/lib/listener/helpers.js:26:69`
    
Then `await Promise.all(contexts.map(c => c.close()));` at line 528 was not reached.
However having this patch, we can reach that section I guess. 


Note: I use `restart: "context"` option

If we use `restart: "session"` option, we can get the  same `TypeError: this.browser.contexts is not a function` error at line 3436. we also need to have a fix for that option.

Reply:
Thank you @kobenguyent , I think this time it works for me without any error.